Week1/homework/js-exercises/logHello.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+{
+  const strEng = 'Hello World!';
+  console.log(strEng);
+
+  const strTur = 'Merhaba Dunya!';
+  console.log(strTur);
+
+  const strDut = 'Hallo Wereld!';
+  console.log(strDut);
+
+  const strNed = 'Hallo Welt!';
+  console.log(strNed);
+
+  const strFra = 'Bonjour le monde!';
+  console.log(strFra);
+
+  const strSpa = 'Hola Mundo!';
+  console.log(strSpa);
+
+  const strChi = '你好，世界!';
+  console.log(strChi);
+
+  const strJap = 'こんにちは世界!';
+  console.log(strJap);
+
+  const strIta = 'Ciao mondo!';
+  console.log(strIta);
+
+  const strRus = 'Привет, мир!';
+  console.log(strRus);
```

Comment:
👍 but this seems to be a case where you could also use a complex data type like arrays or objects and avoid creating so many variables. Something like:

```
const languages = {
  spa: 'Spanish translation',
  ned: 'Nederlands translation',
  // and so on
};

or

const languages = ['Spanish translation', 'Nederlands translation', ..and so on];
```